fftvis/beams.py

```diff
@@ -101,3 +101,98 @@ def get_apparent_flux_polarized(beam: np.ndarray, flux: np.ndarray):  # pragma:
         beam[0, 1, isrc] = i01 * flux[isrc]
         beam[1, 0, isrc] = np.conj(i01) * flux[isrc]
         beam[1, 1, isrc] = i11 * flux[isrc]
+
+@nb.jit(
+    nopython=True,
+    parallel=False,
+    nogil=False
+)
+def get_apparent_flux_polarized(beam: np.ndarray, coherency: np.ndarray): # pragma: no cover
```

Comment:
I think we need to write some units tests for this function (could be as easy as comparing to a simpler numpy-based function that uses `np.dot`)